test/support/constants.js

```diff
@@ -2,6 +2,7 @@ Cypress.env({
     // NOTE: The token used is a readonly token accessing public data in a demo enterprise.
     ACCESS_TOKEN: 'S8wjvjOL9GEK5VtXsQNVMOwSrx1g55oC',
     FILE_ID_ARCHIVE: '589625563707',
+    FILE_ID_ARCHIVE_LONG: '611800378713',
```

Comment:
It looks like we have `FILE_ID_DOC_LARGE`. Can we rename to `FILE_ID_ARCHIVE_LARGE`, as well?